src/bvh/collision_object.cpp

```diff
@@ -115,7 +108,7 @@ namespace bvh
   void collision_object::set_entity_data_impl( const void *_data, std::size_t _element_size )
   {
     const int rank = static_cast< int >( ::vt::theContext()->getNode() );
-    const auto od_factor = m_impl->overdecomposition;
+    const auto od_factor = static_cast< std::size_t >( m_impl->overdecomposition );
```

Comment:
Maybe it's better to just make `collision_object::impl::overdecomposition` unsigned (it's currently an `int`). Does it ever make sense for overdecomposition to be negative?

Reply:
No it should never be negative, we can make it size_t

Reply:
Done.